docs/en/faq/general.md

```diff
@@ -11,9 +11,9 @@ Distributed sorting is one of the main causes of reduced performance when runnin
 
 Most MapReduce implementations allow you to execute arbitrary code on a cluster. But a declarative query language is better suited to OLAP in order to run experiments quickly. For example, Hadoop has Hive and Pig. Also consider Cloudera Impala or Shark (outdated) for Spark, as well as Spark SQL, Presto, and Apache Drill. Performance when running such tasks is highly sub-optimal compared to specialized systems, but relatively high latency makes it unrealistic to use these systems as the backend for a web interface.
 
-## What to do if I have a problem with encodings when using Oracle through ODBC? {#oracle-odbc-encodings}
+## What If I Have a Problem with Encodings When Using Oracle Through ODBC? {#oracle-odbc-encodings}
```

Comment:
maybe we should introduce some different formatting here so questions are not headers

Reply:
It looks that its a common point. For example:
https://cloud.yandex.com/docs/storage/qa (and all other cloud documents)
https://cloud.google.com/storage/docs/faq
Amazon do the same.

Reply:
@BayoNet looks like https://python-markdown.github.io/extensions/definition_lists/

Reply:
Wrong link?